src/sif/autocost.cc

```diff
@@ -391,6 +391,10 @@ bool AutoCost::Allowed(const baldr::DirectedEdge* edge,
                        const uint64_t current_time,
                        const uint32_t tz_index,
                        bool& has_time_restrictions) const {
+
+  if (tile->IsClosedDueToTraffic(edge))
```

Comment:
we need to add this to many more costings, all the others in this file as well as all motorcycle, motorscooter and truck